modules/deploy/pages/deployment-option/cloud/gcp-private-service-connect.adoc

```diff
@@ -0,0 +1,428 @@
++
+[,bash]
+----
+CLUSTER_ID=<cluster_id>
+----
+
+. Update the Redpanda Cloud Agent IAM role. This step is required only for clusters with customer-managed resources.
++
+To allow the agent to create and manage the service attachment, add the following permissions to its IAM role:
++
+[,bash]
+----
+compute.forwardingRules.use
+compute.regionOperations.get
+compute.serviceAttachments.create
+compute.serviceAttachments.delete
+compute.serviceAttachments.get
+compute.serviceAttachments.list
+compute.serviceAttachments.update
+compute.subnetworks.use
+----
+
+. Make a https://docs.api.redpanda.com/#patch-/v1beta1/clusters/-cluster.id-[`PATCH /v1beta1/clusters/{cluster.id}`] request to update the cluster to include the newly-created PSC NAT subnet.
++
+Set the following environment variable:
++
+--
+* `PSC_NAT_SUBNET_NAME`: The name of the PSC NAT subnet. Use the fully-qualified name, for example `"projects/<PROJECT>/regions/<REGION>/subnetworks/<SUBNET-NAME>"`.
+--
++
+[,bash]
+----
+ACCEPT_LIST='[]'
+PSC_NAT_SUBNET_NAME='<PSC NAT subnet name>'
+CLUSTER_PATCH_BODY=`cat << EOF
+{
+    "customer_managed_resources": {
+        "gcp": {
+            "psc_nat_subnet_name": "$PSC_NAT_SUBNET_NAME"
+        }
+    }
+}
+EOF`
+curl -v -X PATCH \
+-H "Content-Type: application/json" \
+-H "Authorization: Bearer $AUTH_TOKEN" \
+-d "$CLUSTER_PATCH_BODY" ${PUBLIC_API_ENDPOINT}/v1beta1/clusters/$CLUSTER_ID
+----
+
+. Make a https://docs.api.redpanda.com/#patch-/v1beta1/clusters/-cluster.id-[`PATCH /v1beta1/clusters/{cluster.id}`] request to update the cluster to enable PSC.
++
+Set the following environment variable:
++
+--
+`ACCEPT_LIST`: a JSON list specifying the projects from which incoming connections will be accepted. All other sources. For example, `[{"source": "consumer-project-ID-1"},{"source": "consumer-project-ID-2"}]`.
+--
++
+[,bash]
+----
+CLUSTER_PATCH_BODY=`cat << EOF
+{
+    "private_link": {
+        "enabled": true,
+        "gcp": {
+            "consumer_accept_list": $ACCEPT_LIST
+        }
+    }
+}
+EOF`
+curl -v -X PATCH \
+-H "Content-Type: application/json" \
+-H "Authorization: Bearer $AUTH_TOKEN" \
+-d "$CLUSTER_PATCH_BODY" ${PUBLIC_API_ENDPOINT}/v1beta1/clusters/$CLUSTER_ID
+----
++
+Wait for the cluster to apply the new configuration (around 15 minutes). The PSC service attachment is available when the cluster update is complete. You can monitor the service attachment creation by running the following `gcloud` command:
++
+[,bash]
+----
+gcloud compute service-attachments list --project '<service-project-ID>'
+----
+
+== Test the connection
+
+You can test the PSC connection from any VM or container in the consumer VPC. If configuring a client isn't possible right away, you can do these checks using `rpk` or curl:
+
+=== Test Kafka API
+
+. Set the following environment variables.
++
+[,bash]
+----
+REDPANDA_BROKERS='<PSC-Kafka-API-seed-URL>'
```

Comment:
@0x5d do we know yet where or how the seed URL can be found in the UI?

Reply:
The design is subject to change, but the latest is that it will be in the "How to connect" section: https://redpandadata.slack.com/archives/C05UXDLB41M/p1711392331097759